src/ukify/ukify.py

```diff
@@ -1394,9 +1399,30 @@ def config_example(self) -> tuple[Optional[str], Optional[str], Optional[str]]:
 
 def apply_config(namespace, filename=None):
     if filename is None:
-        filename = namespace.config
+        if namespace.config:
+            # Config file specified, turn it into a pathlib.Path
+            filename = pathlib.Path(namespace.config)
+        else:
+            # If there is no config file provided, try to look for default configs and load the first found.
+            for config_dir in DEFAULT_CONFIG_DIRS:
+                config_path = pathlib.Path(config_dir).joinpath(DEFAULT_CONFIG_FILE)
+                if config_path.is_file():
```

Comment:
I think `.exists()` is better. We shouldn't really care whether it's a regular file or not.

Reply:
Path.exists() returns true even if it is a directory and it's kinda hard to read INI config from a directory in this setup, so figured it should specifically check whether it is a file/symlink to a file. Unless, you reckon reading multiple files in a directory, something like `ukify.conf.d`, is a good idea. Others do it so I guess we can do that one, although can't really think of any use case for that.

Reply:
I think it should just fails with a IsADirectoryError in that case. But this is not very important, `is_file` is good enough.